### PR TITLE
feat : added modal & other utilities

### PR DESCRIPTION
**Pull request by abhishek17551**
Screenshot (128)
Screenshot (129)
Screenshot (130)
Screenshot (126)
Screenshot (127)

### CODE REVIEW

File: pages/components/modal-utils/modal.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.modal-container {
+    min-width: 20rem;
+    max-width: 27rem;
+    margin: auto;
+    border: 3px solid var(--primary-color-dark);
+    height: 10rem;
+    display: flex;
+    flex-direction: column;
```
Comment (haarsh24) [inline pages/components/modal-utils/modal.css:8]: These properties are used multiple time , so you can declare them and use it again and again.

**Review by haarsh24 — APPROVED** [review]

File: pages/components/modal-utils/modal.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                <a href="./modal.html" class="padding-m">Modal</a>
+                <a href="./other-utils.html" class="padding-m">Other Utilities</a>
+            </ul>
+      
+        
+            <div class="page-wrapper">
+                <h1>Modals</h1>
+                <p class="mg-t-s">Copy the styles of different modals and use it in your code</p>
+                <div>
+                    <h3 class="mg-t-s">Modal</h3>
```
Comment (akash-1318) [inline pages/components/modal-utils/modal.html:47]: I don't think you should use class names like this mg-t-s , i am net getting any idea what's this is for try to give some meaningfull name.

Comment (abhishek17551) [reply pages/components/modal-utils/modal.html:47]: its a short notation for class margin-top-small.

File: pages/components/modal-utils/modal.css (hunk 1 above)
Comment (abhishek17551) [reply pages/components/modal-utils/modal.css:8]: will make the necessary changes.

**Pull request merged**